src/pudl/transform/eia861.py

```diff
@@ -692,63 +712,57 @@ def _clean_nerc_add_rows(df, idx_cols, idx_no_nerc):
+    nerc_df.loc[nerc_df.utility_id_eia == 55959, 'nerc_region'] = 'MRO'
+
     # Make all values upper-case
     # Replace all NA values with UNK
-    # Make nerc values into lists to see how many separate values are stuffed into one row (ex: SPP & ERCOT)
+    # Make nerc values into lists to see how many separate values are stuffed into one row (ex: 'SPP & ERCOT' --> ['SPP', 'ERCOT'])
     nerc_df = (
         nerc_df.assign(
             nerc_region=(lambda x: (
                 x.nerc_region
                 .str.upper()
                 .fillna('UNK')
-                .str.findall(r'[A-Z]+'))),
-            multiple_nercs=(lambda x: (
-                x.nerc_region.apply(lambda x: len(x) > 1)))
+                .str.findall(r'[A-Z]+')))
         )
     )
 
-    # Needed nerc_regions as a list to calculate multiple_nercs, but now can convert nerc lists into
-    # comma-separated strings (can't use same col name twice in one assign)
-    nerc_df['nerc_region'] = nerc_df['nerc_region'].str.join(',')
-
-    # Make dfs of multiple vs. single nerc values
-    nerc_multiples = (
-        nerc_df[nerc_df['multiple_nercs']].copy()
-        .drop('multiple_nercs', axis=1)
-    )
-    nerc_singles = (
-        nerc_df[~nerc_df['multiple_nercs']].copy()
-        .drop('multiple_nercs', axis=1)
-    )
-
-    new_nerc_rows = (
-        nerc_multiples
-        .set_index(idx_no_nerc)
-        .stack()
-        .str.split(',', expand=True)
-        .stack()
-        .unstack(-2)
-        .reset_index(-1, drop=True)
-        .reset_index()
-    )
+    # Record a list of the reported nerc regions not included in the recognized regions list (these eventually become UNK)
+    nerc_col = nerc_df['nerc_region'].tolist()
+    nerc_list = list(set([item for sublist in nerc_col for item in sublist]))
+    non_nerc_list = [
+        nerc_entity for nerc_entity in nerc_list if nerc_entity not in RECOGNIZED_NERC_REGIONS + list(NERC_SPELLCHECK.keys())]
+    logger.info(
+        f'The following reported NERC regions are not currently recognized and become UNK values: {non_nerc_list}')
 
-    # Recombine old nerc single rows with new nerc expanded rows
-    nerc_df = nerc_singles.append(new_nerc_rows)
+    # Function to turn instances of 'SPP_UNK' into 'SPP' (or other recognized nerc regions)
+    def _remove_unk_multiples(entity_list):
+        if ((len(entity_list) > 1) & ('UNK' in entity_list)):
+            entity_list.remove('UNK')
+        return entity_list
 
-    # Anything that isn't in the official list of nerc_regions is 'UNK'
+    # Go through the nerc regions, spellcheck errors, delete those that aren't recognized, and piece them back together
+    # (with _ separator if more than one recognized)
     nerc_df['nerc_region'] = (
         nerc_df['nerc_region']
-        .apply(lambda x: x if x in RECOGNIZED_NERC_REGIONS else 'UNK')
+        .apply(lambda x: [i if i not in NERC_SPELLCHECK.keys() else NERC_SPELLCHECK[i] for i in x])
```

Comment:
this one might be cleaner and faster to use [replace](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.replace.html). It looks like your `NERC_SPELLCHECK` is already perfectly set up for it

Reply:
Hmm that'd be great! I'm not sure I see how to incorporate replace, though. It's a little tricky because it's a column of lists.

Reply:
oooh! I forgot about the column of lists thing! ignore this then :-)